src/bin/tikv-server.rs

```diff
@@ -194,6 +194,21 @@ fn get_rocksdb_option(matches: &Matches, config: &toml::Value) -> RocksdbOptions
     let wal_recovery_mode = util::config::parse_rocksdb_wal_recovery_mode(rmode).unwrap();
     opts.set_wal_recovery_mode(wal_recovery_mode);
 
+    let enable_statistics = config.lookup("rocksdb.enable-statistics")
```

Comment:
why use config lookup here? I think we should support get_bool_value here. 
/cc @overvenus